### DISCUSSION TITLE
battery wire connector

### DISCUSSION THREAD

**Discussion by 4perros** [bitcraze/discussions#437] [Q&A] [answered]
hello
i have a question regarding the battery connector. The wire that goes into the crazyflie broke. i managed to get some but the problem is that the wire is thinner than the original one that also has a silicon insulation, also i checked the number of starnds in the original wire and it has about 11 strands, the connector i bought has only 7 strands and the insulation is not silicon. I would like to know where I can purchase the type of connector wire used on the crazyflie.
regards
rodrigo

**Comment by Jarvis-X** [comment, marked as answer]
Hi, Rodrigo,

On [the datasheet of cf2.1](https://www.bitcraze.io/documentation/hardware/crazyflie_2_1/crazyflie_2_1-datasheet.pdf), Section 3. Onboard microcontrollers, it says the onboard LiPo charger supports up to the current output of 980mA. Since the only outlet connecting the board and the LiPo is the power cable, it means the cable can sustain a current no less than 980 mA. I would suggest finding cables that have a current capacity of 1A at the very least. The insulation should also be able to contain said current.

Jiawei

**Reply by tobbeanton** [reply]
The wire to the CF2.1 connector is a 24awg silicon coated copper wire. Search for this and it shouldn't be to hard to find. It will work with thinner wires as well but the added resistance in the wire will create a bit more losses.

**Reply by 4perros** [reply]
hi
thank you for the information.
regards
rodrigo
